### PR TITLE
📖 Create a yaml issue template for a Bento component tracker

### LINKED ISSUE #34739: Provide an issue template for "Bento tracking issue"
### Description

It would be very helpful to @ampproject/wg-bento contributors to have a "Bento tracking issue" template. We already have many existing such issues following identical templates which are manually filled in and filed. 

## Template requirements
- Title in the format of `[amp-component-name:version] Bento tracking issue`
- High-level requirements
  This is a matrix of considerations that the component must address. There are three columns: `Consideration`, `Ready when`, and `Status`. This ideally contains links specific to the `amp-component-name`'s documentation on amp.dev and existing accessibility issues filed with that component's label (if existent).
- Component compatibility
  - Feature checklist, such as existing attributes, actions, events, and analytics events that exist in the classic version and whether/how they are supported in the new Bento version. This may need to be manually copied from the existing component `md` file.
  - Migration notes - If there are any API changes that prevents document authors from upgrading a classic version to the new Bento version without checking anything else, that should go in this section.
- Open tasks
  - Check list for issue author or code reviewers to maintain

## Example issues
#28281, #31366, #30443, or anything in https://github.com/ampproject/amphtml/projects/100

### Alternatives Considered

- We could continue manually creating these issues, but given there are dozens of components to convert, we'd get a lot out of being able to remove some overhead with this template.
- We could [create a link to file an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/about-automation-for-issues-and-pull-requests-with-query-parameters) and use query parameters to fills the issue body with content. A downside to this is that it's not officially findable via "Create an issue" in `amphtml`, but perhaps this could also be an upside given that this type of issue may be more niche and could be linked from @ampproject/wg-bento or onboarding docs instead.

### Additional Context

I unfortunately can't do it myself since I don't have access to the alpha feature 😅  Relevant to #34400 cc/ @rsimha

### PR DESCRIPTION
**Pull request by rsimha**
**Screenshots:**

image
image
image

Fixes #34739

### CODE REVIEW

**Comment by rsimha** [comment]
@caroqliu Here's a first draft of the issue template you requested. If you add comments / suggestions to the raw yaml, I can apply them and update the screenshots in the description.

Once you're satisfied, we can merge this PR at which point you can directly make edits to the yaml file.

👍

File: .github/ISSUE_TEMPLATE/bento-component-tracker.yml
Comment (caroqliu) [inline .github/ISSUE_TEMPLATE/bento-component-tracker.yml:73]: Relevant information for document authors using the classic version of the component.

File: .github/ISSUE_TEMPLATE/bento-component-tracker.yml
Comment (caroqliu) [inline .github/ISSUE_TEMPLATE/bento-component-tracker.yml:13]:
```suggestion
        - See past [tracking issues](https://github.com/ampproject/amphtml/issues?q=is%3Aopen+label%3A%22WG%3A+bento%22+%22bento+tracking+issue%22) for information about other Bento components.
```

File: .github/ISSUE_TEMPLATE/bento-component-tracker.yml
Comment (caroqliu) [inline .github/ISSUE_TEMPLATE/bento-component-tracker.yml:12]:
```suggestion
        - See AMP's [documentation](https://github.com/ampproject/amphtml/blob/main/docs/building-a-bento-amp-extension.md) to learn about how to write a Bento component.
        - See additional documentation for considerations when [Building a Bento Video Player](https://github.com/ampproject/amphtml/blob/main/docs/building-a-bento-video-player.md).
```

File: .github/ISSUE_TEMPLATE/bento-component-tracker.yml
Comment (caroqliu) [inline .github/ISSUE_TEMPLATE/bento-component-tracker.yml:34]:
```suggestion
        | Consideration           | Ready When      | Status  |
        | ----------------------- | --------------- | ------- |
        | Component Compatibility | Feature checklist is covered | ✅ / ✖️ |
        | Testing                 | Unit tests and pre-existing e2e tests pass | ✅ / ✖️ |
        | Internationalization    | <fill this out> | ✅ / ✖️ |
        | Analytics               | <fill this out> | ✅ / ✖️ |
        | Accessibility           | Audit performed | ✅ / ✖️ |
        | Page Experience         | Audit performed | ✅ / ✖️ |
        | Documentation           | Written | ✅ / ✖️ |
        | Storybook               | Samples written in Preact and AMP | ✅ / ✖️ |
```

File: .github/ISSUE_TEMPLATE/bento-component-tracker.yml
Comment (caroqliu) [inline .github/ISSUE_TEMPLATE/bento-component-tracker.yml:68]:
```suggestion
        ### Analytics Events
        - [ ] `analytics event 1`
        - [ ] `analytics event 2`
        - [ ] `analytics event 3`
```

File: .github/ISSUE_TEMPLATE/bento-component-tracker.yml
Comment (caroqliu) [inline .github/ISSUE_TEMPLATE/bento-component-tracker.yml:73]:
```suggestion
      description: Relevant information for document authors using the classic version of the component.
```

**Review by caroqliu — APPROVED** [review]
This looks amazing, thank you Raghu! Left a few revisions inline.

**Comment by rsimha** [comment]
All edits made. Did some additional clean up to standardize link formatting, fix typos, and validate the form (all fields ought to be required). Will merge once CI is green.

**Pull request merged**
